src/web/server/render.ts

```diff
@@ -3,12 +3,14 @@ import { extract as extractNAV } from '@root/src/model/extract-nav';
 
 import { document } from '@root/src/web/server/document';
 import { validateAsCAPIType } from '@root/src/model/validate';
+import { addDropCaps } from '@root/src/model/add-dropcaps';
 import { extract as extractGA } from '@root/src/model/extract-ga';
 import { bodyJSON } from '@root/src/model/exampleBodyJSON';
 
 export const render = ({ body }: express.Request, res: express.Response) => {
     try {
-        const CAPI = validateAsCAPIType(body);
+        const withDropCaps: CAPIType = addDropCaps(body);
```

Comment:
I think maybe having a hop into a `enhanceCAPI` file, then then collates all the hacks might make it easier to reason with as render.ts could get large. But yeah, not urgent for this PR.

Reply:
I had the same thought 👍